charter.md

```diff
@@ -1,47 +1,31 @@
-these HTTP versions.
-
-IP multicast is out of scope. Extensions providing generic capabilities that might include IP multicast are acceptable
-on the basis of their application to other use cases. For example, a generic IP proxy might be used for both unicast
-and multicast communication. For such generic capabilities, designs will not explicitly preclude multicast, but they
-will not focus on multicast-specific features.
-
-Impacts on address migration, NAT rebinding, and future multipath mechanisms of QUIC are not anticipated. However, the
-working group should document these impacts, or those of any other QUIC developments, if they arise.
-
-The group will coordinate closely with other working groups responsible for maintaining relevant protocol extensions,
-such as HTTPBIS, QUIC, or TLS. It will also coordinate closely with ICCRG and TSVWG on congestion control and loss
-recovery considerations, and intarea for IP Proxying.
-
-When these deliverables are complete, the working group will either recharter to reflect additional work or close.
\ No newline at end of file
+Many network topologies lead to situations where transport protocol proxying is beneficial—for example, enabling endpoints to communicate when end-to-end connectivity is not possible, or applying additional encryption such as in VPNs. Proxying can also improve client privacy by hiding client IP addresses from target servers. Existing proxying technologies (e.g., SOCKS and HTTP CONNECT) have limitations: SOCKS lacks encryption, and HTTP CONNECT is limited to TCP.
+
+The MASQUE working group has developed mechanisms that allow configuring and concurrently running multiple proxied stream- and datagram-based flows inside an HTTP connection. These include CONNECT-UDP and CONNECT-IP, collectively known as MASQUE. MASQUE leverages HTTP semantics, multiplexes flows over streams, uses a unified congestion controller, encrypts flow metadata, and enables unreliable delivery suitable for UDP and IP-based applications.
+
+The MASQUE working group is now in the process of completing its remaining deliverables:
+
+ 1. A mechanism for proxying Ethernet over HTTP
+ 2. An extension to CONNECT-UDP that is specific to proxying QUIC
+ 3. An extension to CONNECT-UDP that allows proxying bound sockets
+ 4. An extension to CONNECT-IP that allows conveying DNS and PREF64 configuration
+ 5. An informational architecture document about MASQUE
+ 6. An extension to CONNECT-UDP for ECN and DSCP support
+ 7. An extension to CONNECT-UDP and / or CONNECT-IP to enable compression
+ 8. A definition of qlog nomenclature for existing MASQUE mechanisms
+
+These documents are being finished in line with MASQUE’s previous charter: 
+
+* The intended status is Standards Track unless otherwise noted, but the WG may downgrade if it believes that is appropriate for the ultimate document maturity level.
```

Comment:
This was useful in the early days of the WG but seems like we can strike this out now. If the intended status is useful, just state that in each of the handful of work items